Runtime/Plugins/GLTFSerialization/Schema/GLTFMaterial.cs

```diff
@@ -173,9 +173,9 @@ public static GLTFMaterial Deserialize(GLTFRoot root, JsonReader reader)
 				}
 			}
 
-			if (material.Extras is not null && 
-				material.Extras.Contains("shaderName") && 
-				material.Extras["shaderName"] is JValue shaderName) {
+			if (material.Extras is JObject extras && 
+				extras.ContainsKey("shaderName") && 
+				extras["shaderName"] is JValue shaderName) {
```

Comment:
With this many checks already (good), this should probably also make sure that the value is actually a string 🤔